### DISCUSSION TITLE
Labels (and counters) export as a csv

### DISCUSSION THREAD

**Discussion by Antiokh** [martinrotter/rssguard#1108] [Ideas]
Hello, thank you for your help with the label creation.
After some amount of time I got the stats and want to do a frequency analysis.

How popular is each one tag. Is there a way to somehow export the label stats?

**Comment by martinrotter** [comment]
Well, directly in RSS Guard this is likely not possible right now.

It would be possible probably with some SQL code. You can open "database.db" file directly in some SQLite tool, for example SQLiteBrowser, then run SQL code in it to get results you want.

**Reply by Antiokh** [reply]
Thanks! It works!
